content/docs/studio/user-guide/projects-and-experiments/explore-ml-experiments.md

```diff
@@ -22,21 +21,26 @@ includes metrics, hyperparameters, and information about datasets and models.
 
 ## Components of a project
 
-The experiments table of the project has different components that show you the
-complete experiment history as well as enable to you to generate plots, compare
-experiments, run new experiments, etc. The major components of the table are:
+The major components of the project experimentation table are:
 
-- [Git history](#git-history) - represents your experimentation history.
-- [Display preferences](#display-preferences) - show/hide branches, commits and
-  columns, and re-arrange the table.
+- [Git history and live metrics](#git-history-and-live-metrics) that show you
+  the complete experimentation history as well as live metrics of running
+  experiments.
+- [Display preferences](#display-preferences) that let you show/hide branches,
+  commits and columns, and re-arrange the table.
 - Buttons to
   [visualize, compare, and run experiments](#visualize-compare-and-run-experiments).
 
-### Git history
+### Git history and live metrics
```

Comment:
We also have a special row of "experiment commit", which was initiated by the user and was not yet committed to the repo/errored out. Such rows do not have any metrics but do have modified params/files.

Reply:
> which was initiated by the user and was not yet committed

The row initiated by the user is committed to Git, right? Do you mean this: `Any live metrics that you send using [DVCLive] are displayed in a special experiment row next to the corresponding Git commit.`

Reply:
No, I actually mean information completely unrelated to live metrics. For our "Run experiment" flow, when the user hits "submit experiment", Studio will try to commit this experiment to the repository, but it happens not immediately. Until this experiment is committed (or if it has an error), we display a different type of row which is neither a part of git history, nor a live experiment. It is a special "experiment commit" row type.

I'm wondering whether we should include it here as well for completeness.

Reply:
Ok. Updated.